specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/virtualWan.json

```diff
@@ -5279,6 +5644,74 @@
         }
       },
       "description": "ExpressRouteConnection list."
+    },
+    "RoutingConfiguration": {
+      "description": "Routing Configuration indicating the associated and propagated route tables for this connection.",
+      "properties": {
+        "associatedRouteTable": {
+          "$ref": "./network.json#/definitions/SubResource",
+          "description": "The resource id RouteTable associated with this RoutingConfiguration."
+        },
+        "propagatedRouteTables": {
+          "$ref": "#/definitions/PropagatedRouteTable",
+          "description": "The list of RouteTables to advertise the routes to."
+        },
+        "vnetRoutes": {
+          "$ref": "#/definitions/VnetRoute",
+          "description": "List of routes that control routing from VirtualHub into a virtual network connection."
```

Comment:
>VirtualHub [](start = 67, length = 10)

isn't this definition referenced by a number of resource types other than virtualHub?  Or is this all considered virtualhub?